src/OrleansRuntime/GrainDirectory/ClientObserverRegistrar.cs

```diff
@@ -117,16 +119,8 @@ internal void ClientDropped(GrainId clientId)
             }
             catch (Exception exc)
             {
-                int actualExceptions = 1;
-                if (exc is AggregateException)
-                {
-                    AggregateException aggregateException = exc as AggregateException;
-                    actualExceptions = aggregateException.InnerExceptions.Count;
-                    exc = aggregateException.InnerExceptions.First();
-                }
                 logger.Error(ErrorCode.ClientRegistrarTimerFailed, 
-                    String.Format("OnClientRefreshTimer has thrown {0} inner exceptions. Printing the first exception:", actualExceptions), 
-                    exc);
+                    String.Format("OnClientRefreshTimer has thrown an exceptions."), exc);
```

Comment:
"now correct" you mean with my change here or do you want me to revert back to the previous code?
I can do either, which ever you prefer.